app/helpers/requests_helper.rb

```diff
@@ -28,15 +28,20 @@ def select_for_multiple_libraries(form, pickup_libraries)
     return unless pickup_libraries.keys.length > 1
     form.select :destination,
                 pickup_libraries_array(pickup_libraries),
-                label: label_for_pickup_libraries_dropdown(pickup_libraries),
-                selected: 'GREEN'
+                { label: label_for_pickup_libraries_dropdown(pickup_libraries), selected: 'GREEN' },
+                'aria-controls' => 'scheduler-text',
+                'data-paging-schedule-updater' => 'true'
```

Comment:
Both aria and data attributes could be written e.g.:

```
aria: { controls: 'scheduler-text' }
```

Since it's only a single value, i don't know if it matters much.